plusdeck/client.py

```diff
@@ -415,8 +410,6 @@ def wait_for(
         def listener() -> None:
             fut.set_result(None)
 
-        self.events.on("state", listener)
```

Comment:
This was a bug! I believe the tests aren't asserting the listener is only called once.